report/sections/sprint1/change-of-framework.tex

```diff
@@ -0,0 +1,32 @@
+\section{Change of framework}
+As previously mentioned, the developers of last year had made the choice of changing the development framework from purely using Android Studio to Xamarin in order to support building the application for iOS.
+However, with the introduction of full-stack teams this lead to a series of issues, as approximately 1/4th of the developers on the GIRAF project this year were using some variation of Linux as their operating system of choice, which Xamarin does not officially support.
+Previously, this had not been a problem, as only the dedicated front-end development group would be working with Xamarin. 
+Only the people in this group would need to be on Windows or MacOS in order to compile the front-end part of the application.
+With full-stack groups, more or less every group would be affected by this problem, having group members who could not compile the front-end of the application, which is the main focus of this year.
+This led to one group spending a lot of time investigating different ways to make the Xamarin project compile on Linux, but as time progressed it was deemed infeasible to implement a series of workarounds to make the project compile.
+Instead, the front-end skill group held a meeting to discuss the pros and cons of making the Xamarin project work versus changing to another framework, which would natively support Linux.
+During the meeting, a series of pros and cons for changing the framework were worked out:
+
+\begin{itemize}
+    \item [\textbf{Pros}]
+    \item The most difficult part to reimplement is authorization
+    \item It is difficult to maintain the Linux compatibility for Linux, as the problems varied between machines
```

Comment:
Ved ikke helt om jeg forstår linux kompatibilitet for linux, er der et blow up her?

Reply:
Yeee, den ene skal være fanatisk lul

Reply:
Xamarin, autocorrect pls